packages/shared/__tests__/describeComponentFrame-test.js

```diff
@@ -12,15 +12,18 @@
 let React;
 let ReactDOMClient;
 let act;
+let jsxDEV;
 
 describe('Component stack trace displaying', () => {
   beforeEach(() => {
     React = require('react');
-    ReactDOMClient = require('react-dom');
+    ReactDOMClient = require('react-dom/client');
     act = require('internal-test-utils').act;
+    jsxDEV = require('react/jsx-dev-runtime').jsxDEV;
   });
 
-  // @gate !enableComponentStackLocations || !__DEV__
+  // @gate !enableComponentStackLocations
+  // @gate __DEV__
```

Comment:
i like it, have you started preferring this over conditionals?

Reply:
Mostly just because the diffs are nicer